src/engine/source/api/src/catalog/catalog.cpp

```diff
@@ -161,7 +161,7 @@ std::optional<base::Error> Catalog::postResource(const Resource& collection,
     if (Resource::Type::COLLECTION != collection.m_type)
     {
         return base::Error {
-            fmt::format("Expected resource type is \"{}\", but got \"{}\"",
+            fmt::format(R"(Expected resource type is "{}", but got "{}")",
```

Comment:
[Coding style, Modern language features](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices)